notification/slack/util.go

```diff
@@ -0,0 +1,36 @@
+package slack
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/target/goalert/config"
+)
+
+type Meta struct {
+	TeamID string `json:"team_id"`
+}
+
+func GetTeamID(ctx context.Context) (*string, error) {
```

Comment:
Make this private and just return a string instead of a pointer. As a helper method, pass the token directly instead of pulling values from context so dependencies/arguments are clear.

```suggestion
func lookupTeamIDForToken(ctx context.Context, token string) (string, error) {
```